pkg/printers/internalversion/describe.go

```diff
@@ -1129,6 +1129,9 @@ func printCSIPersistentVolumeSource(csi *api.CSIPersistentVolumeSource, w Prefix
 		"    VolumeHandle:\t%v\n"+
 		"    ReadOnly:\t%v\n",
 		csi.Driver, csi.VolumeHandle, csi.ReadOnly)
+	for key, value := range csi.VolumeAttributes {
```

Comment:
There should also be a "    VolumeAttributes:\n" line, and then I think the attributes should be printed out after that with one more indentation level.